figures/tasks.py

```diff
@@ -17,7 +18,7 @@
 from figures.helpers import as_course_key, as_date
 from figures.pipeline.course_daily_metrics import CourseDailyMetricsLoader
 from figures.pipeline.site_daily_metrics import SiteDailyMetricsLoader
-
+import figures.sites
```

Comment:
@johnbaldwin Why do you prefer to import the whole `figures.sites` helper rather than importing just the function that is gonna be used here?

Reply:
@melvinsoft I learned it is easier to use `mock` when the test class and the production code use the same module name so that when the production code is called from test that uses `figures.sites` (or any full namespace for that matter), then you can use `mock.patch`

Example:
```
with mock.patch('figures.settings.env_tokens', self.env_tokens):
    do_something(x)
```

When I do `from x import y` the production code doesn't seem consistent in using the mocked value. There are a number of examples on the web mentioning this. Here's one SO: https://stackoverflow.com/questions/11351382/mock-patching-from-import-statement-in-python

Let me know if I need to explain this more

Reply:
@johnbaldwin Thanks for the explanation! Makes sense.

Reply:
@johnbaldwin for your reference, it you'd like to do an `from import figures.sites import funky_function` in `figures/tasks.py`.

You should patch it like below:

```
with mock.patch('figures.tasks.funky_function`, self.funky_function):
  do_something(x)
```

It's confusing, but this is how it works.